tensorboard/webapp/metrics/views/right_pane/right_pane_test.ts

```diff
@@ -402,7 +402,7 @@ describe('metrics right_pane', () => {
 
         const el = fixture.debugElement.query(By.css('.linked-time'));
         const [enabled] = el.queryAll(By.css('mat-checkbox input'));
-        expect(enabled.nativeElement.ariaChecked).toBe('false');
+        expect(enabled.nativeElement.checked).toBe(false);
```

Comment:
nit: We typically use toBeFalse()/toBeTrue() for these.

Reply:
Done.